boards/BMS/Src/Io/Io_LTC6813/Io_SharedLTC6813.c

```diff
@@ -233,48 +242,48 @@ void Io_NewLTC6813_InitSpiHandle(SPI_HandleTypeDef *spi_handle)
         spi_handle, SPI2_NSS_GPIO_Port, SPI2_NSS_Pin, SPI_TIMEOUT_MS);
 }
 
-uint16_t Io_NewLTC6813_CalculateRegGroupPec15(uint8_t *data_buffer)
+uint16_t Io_SharedLTC6813_CalculateRegGroupPec15(uint8_t *data_buffer)
 {
-    return Io_NewLTC6813_CalculatePec15(
+    return Io_SharedLTC6813_CalculatePec15(
         data_buffer, NUM_REG_GROUP_PAYLOAD_BYTES);
 }
 
-void Io_NewLTC6813_PackPec15(uint8_t *tx_data, uint8_t num_bytes)
+void Io_SharedLTC6813_PackPec15(uint8_t *tx_data, uint8_t num_bytes)
 {
     const uint16_t cfg_reg_a_pec15 =
```

Comment:
I think naming the var `cfg_reg_a_pec15` is a bit unclear, simply naming it `pec15` would make more sense

Reply:
ah yea, good catch. looks like some leftover var names i can fix this.